examples/absorbed_SynIC.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+import numpy as np
+import astropy.units as u
+from astropy.io import ascii
+
+import naima
+
+## Model definition
+
+from naima.models import InverseCompton, Synchrotron, ExponentialCutoffPowerLaw, BrokenPowerLaw, EblAbsorptionModel
+
+# Define the redshift of the source, and absorption model
+redshift = 0.06 * u.dimensionless_unscaled
+EBL_transmitance = EblAbsorptionModel(redshift, 'Dominguez')
```

Comment:
Can you move the definition of these inside the function as with the other models defined? That makes the function more portable if somebody wants to reuse it.